src/config/workload.rs

```diff
@@ -582,3 +641,43 @@ impl Ratelimit {
         }
     }
 }
+
+#[derive(Clone, Copy, Deserialize)]
+pub struct StoreCommand {
+    verb: StoreVerb,
+    #[serde(default = "one")]
+    weight: usize,
+}
+
+impl StoreCommand {
+    pub fn verb(&self) -> StoreVerb {
+        self.verb
+    }
+
+    pub fn weight(&self) -> usize {
+        self.weight
+    }
+}
+
+#[derive(Clone, Deserialize, Copy, Debug, Ord, Eq, PartialOrd, PartialEq, Hash)]
+#[serde(rename_all = "snake_case")]
+pub enum StoreVerb {
+    /// Sends a `PING` to the server and expects a `PONG`
+    /// * Ping: `PING`
+    /// * RESP: `PING`
+    Ping,
+
+    /*
+     * KEY-VALUE
+     */
+    /// Read the value for one a key.
+    /// * Momento: `get`
+    Get,
+    /// Set the value for a key.
+    /// * Momento: `set`
+    Set,
```

Comment:
@brayniac something we were discussing is having a specific `Put` verb used instead of `Set`, since we are technically calling `put` for Momento. This would also mean adding a `PUT` metric, any oppositions to this?

Reply:
Addressed in https://github.com/iopsystems/rpc-perf/pull/240/commits/c16805047a751de5001e69492295afa893eae5b2